fix

SzymonKomar opened the pull request.

SzymonKomar committed:
fix

rafallakomiec commented:
Jak masz li w którym są tel i email to na 3 karcie ok, choć można byłoby to przemyśleć, bo trochę masło maślane, ale da się odszukać. Natomiast na tym samym poziomie w karcie 1 i 2 masz contacts_item podmień na contacts__contacts-item bo wyżej już masz zapiętą taką klasę na ul'u


Jak masz guziki to transition z :hoverów daj sobie do podstawowej klasy. Bo teraz animacja przy zejściu schodzi twardo, a tak będzie wejście i zejście ok.
Ogólnie wizualnie wychodzi ok.
Chyba to jest najważniejsze. A wysunięcie kart można różnie zrobić, to nie będę szukać dziury w całym. ;)

SzymonKomar committed:
feature

Dodałem mapy (Natomiast są dość proste) oraz możliwość użycia przycisku feature do przejścia do strony na wikipedi franczyza

SylviaStachera committed:
Merge branch 'dev' into feature/contacts
